### PR TITLE
Use ndarray::Array2 as data format

### PR DESCRIPTION
**Pull request by trobanga**
Add R/W file example

### CODE REVIEW

File: tests/rust/.gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created by https://www.toptal.com/developers/gitignore/api/rust
```
Comment (Andrey1994) [inline tests/rust/.gitignore:1]: maybe I am missing smth but in this folder, there is only .gitignore and cargo.toml, did you forgot to add smth?

File: rust-package/brainflow/examples/read-write-file.rs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    let mut tmp_dir = env::temp_dir();
+    tmp_dir.push("brainflow_tests");
+    tmp_dir.push("rust");
+    fs::create_dir_all(&tmp_dir).unwrap();
+    tmp_dir.push("read-write_file.csv");
+
+    let filename = tmp_dir.to_str().unwrap();
+
+    dbg!(&data);
+    // let data = data
```
Comment (Andrey1994) [inline rust-package/brainflow/examples/read-write-file.rs:29]: rm comment

File: docs/UserAPI.rst [hunk 3]
```diff
@@ -149,3 +149,8 @@ Example:
 
 .. literalinclude:: ../julia-package/brainflow/test/serialization.jl
    :language: julia
+
+Rust API Reference
```
Comment (Andrey1994) [inline docs/UserAPI.rst:153]: lets add docs as the latest step, in the current master its removed for now

File: .github/workflows/run_windows.yml [hunk 4]
```diff
@@ -120,6 +120,11 @@ jobs:
       uses: julia-actions/setup-julia@v1
       with:
         version: 1.3.1
+    - name: Build Rust Test
```
Comment (Andrey1994) [inline .github/workflows/run_windows.yml:123]: all examples are in another folder what does it test\build?

File: tests/rust/.gitignore (hunk 1 above)
Comment (trobanga) [reply tests/rust/.gitignore:1]: I started with that branch yesterday and thought I could merge master into it...which was a bad idea. This folder should not exist...that's why I closed the PR. I'll make a new one

**Pull request closed without merging**